src/vue/mixins/offer-maker.mixin.js

```diff
@@ -79,10 +78,14 @@ export default {
 
         await Sdk.horizon.transactions.submitOperations(operation)
 
-        Bus.success(globalize('offer-creation-form.success'))
+        Bus.success('offer-creation-form.success')
       } catch (error) {
-        if (error instanceof errors.BadRequestError) {
-          Bus.error(parseError(error))
+        if (
+          error instanceof errors.TransactionError &&
+          // eslint-disable-next-line
+          error.resultCodes.operations.includes(TRANSACTION_ERROR_CODES.opCrossSelf)
```

Comment:
looks awkward. Why don't write the `includes` method for `TransactionError`?

Reply:
and expose the error codes directly from sdk

Reply:
Don't understand what do you mean. `TransactionError` returns operations codes right now and in this line I only check what exactly operation code I got and in the result of this check, I show an appropriate error message.

Reply:
you can not only return the operation codes, but also expose the method of `TransactionError` that will try to find the op_code in the result array

I mean instead of doing 

```
error.resultCodes.operations.includes(op_code)
```

i want to do

```
error.includes(op_code)
```